### DISCUSSION TITLE
[BUG] Mirth 4.3.0 Error in when on Dashboard - om.mirth.connect.client.core.EntityException:servlet":"org.glassfish.jersey.servlet.ServletContainer

### DISCUSSION THREAD

**Discussion by Chuck-Busch** [nextgenhealthcare/connect#5777] [Q&A]
**Describe the bug**
A new install of Mirth 4.3.0 on Windows 11 professional for home use.  I am getting an error message about every 20 seconds when I am in the Dashboard screen.  The error begins with Method failed: HTTP/1.1 500 Server Error and ends with, "Caused by: com.mirth.connect.client.core.EntityException: {
"servlet":"org.glassfish.jersey.servlet.ServletContainer-294d6733",
"message":"Request failed.",
"url":"/api/4.3.0/channels/statuses/initial",
"status":"500"
}"

I have tried uninstalling and re-installing.

I am also getting this error in the Server Log

[2023-05-08 22:01:57.610]  ERROR  (com.mirth.connect.model.converters.ObjectXMLSerializer:264): com.thoughtworks.xstream.converters.ConversionException: No converter available
---- Debugging information ----
message             : No converter available
type                : java.util.Collections$EmptySet
converter           : com.mirth.connect.model.converters.MigratableConverter
message[1]          : Unable to make field private static final long java.util.Collections$EmptySet.serialVersionUID accessible: module java.base does not "opens java.util" to unnamed module @1cf44b49
converter[1]        : com.thoughtworks.xstream.converters.reflection.ReflectionConverter
-------------------------------

**Comment by ab-20-m** [comment]
The required configuration changes were not applied for Java 9+. See: [Running Mirth Connect in Java 9 or Greater](https://github.com/nextgenhealthcare/connect#5-running-mirth-connect-in-java-9-or-greater).

See also: #5391 #5392 #5577

**Comment by pacmano1** [comment]
Closed, user appeared that they might have missed the documetnation on java 9+ required vmoptions.

**Comment by Chuck-Busch** [comment]
I am still getting the error.  Here are some more details:

![image](https://github.com/nextgenhealthcare/connect/assets/108025827/aa19680e-5fff-427c-b026-214fdba56b97)

I have copied the contents from \docs\mcservice-java9+.vmoptions to both mcserver.vmoptions and mcservice.vmoptions.
I have restarted Mirth Connect Administrator
I only get the error in the Mirth Connect Dashboard
I get it every 20 seconds

Here is the Error
![image](https://github.com/nextgenhealthcare/connect/assets/108025827/7ad0c053-678f-44e4-8f3b-263a5ce13dc3)

Here is the entry in the Server Log
![image](https://github.com/nextgenhealthcare/connect/assets/108025827/8e9d23e2-3345-433b-9ab0-b7cc06c089ae)

**Comment by pacmano1** [comment]
You have to restart the server process.   I replicated your set up with no issues.

**Comment by pacmano1** [comment]
Going to move this to the discussion area for now.

**Comment by Chuck-Busch** [comment]
The restart of the server process worked.  Thanks
